### ISSUE TITLE
US Exchange Support

### ISSUE THREAD

**Issue by spenadel** [taoteh1221/Slideshow_Crypto_Ticker#35]
Does the project support US exchanges such as the NASD?  If so, what's the exchange_markets identifier for NASD, NYSE?

Thank you

**Comment by taoteh1221** [comment]
@spenadel No, see README.txt for the list of supported crypto-only exchanges. Currently supported are: Binance / Coinbase Pro / Kraken / Kucoin / Bitstamp / Bitfinex / HitBTC / OKex / LoopRing / Bitmart / Gate.io

**Comment by spenadel** [comment]
Thanks for the response.  I did see the supported exchanges but still wanted to ask ;)

**Comment by taoteh1221** [comment]
:) I only do crypto assets in my apps. Not a stocks guy, never really have been.

**Comment by spenadel** [comment]
Understood.  Any thought around modifying the program to reflect price charts and/or cost basis to reflect profit/loss?

**Comment by taoteh1221** [comment]
The only thing slated to be added for trend data is the option to show 1h / 24hr / 7d / 30d price change percentage (per market), with coingecko data: https://github.com/taoteh1221/Slideshow_Crypto_Ticker/blob/main/DOCUMENTATION-ETC/TODO.txt#L70

**Comment by taoteh1221** [comment]
It's a big challenge to fit data onto small lcd screens (primary use case of this app), without doing overkill and squeezing too much data into a tiny space. Have to keep it minimal to avoid confusion.

**Comment by hapklaar** [comment]
[btcticker](https://github.com/llvllch/btcticker) did a nice job of a layout with trend graph on a small screen. Could this be something to incorporate into your project?

**Comment by spenadel** [comment]
That's what I was referring to

**Comment by taoteh1221** [comment]
That looks pretty cool. Honestly not sure how much time I'll have to look into something like this and possibly implement anytime soon. But maybe at some point in the future. Thanks very much for the feedback, I appreciate it.
